apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -561,14 +630,15 @@ export function WriteInsAdjudicationScreen(): JSX.Element {
               )}
             </div>
 
-            <div>
+            <div style={{ marginTop: '1rem' }}>
```

Comment:
Why move this down? It looks a little odd to have a gap

Reply:
Not intentional!